### ISSUE TITLE
Protocol says return "good" on success

### ISSUE THREAD

**Issue by bigon** [kbabioch/php-ddns#2]
Hello,

The dyndns protocol says that on success the server should return "good", but here it's returning "success"

This breaks the integrated "dyndns" feature of my ISP router

https://help.dyn.com/remote-access-api/return-codes/

**Comment by kbabioch** [comment]
Does it work when you change `success` to `good` in this line:

https://github.com/kbabioch/php-ddns/blob/4af5251ac906b565e6cb22679a23c64f3142ada5/update.php#L170-L175

I think I took `success` from the output of `nsupdate` and it worked for me just fine, but I guess it depends on the client. Should be a trivial change, but let's first verify that it indeed fixes your problem.

**Comment by bigon** [comment]
Yes yes changing that line to "good" is indeed working (I was not clear enough)

**Comment by bigon** [comment]
Hello @kbabioch 

Is that really fixed? I still see the ` echo 'success'; ` in the code:

https://github.com/kbabioch/php-ddns/blob/940263396124cfbc196c15f14beecde420eced23/src/update.php#L262-L270

**Comment by kbabioch** [comment]
> Is that really fixed? I still see the `echo 'success';` in the code:

Thank you for noticing and letting me know. Slipped through, somehow. Should be fixed with #4 now.
